compiler/src/dotty/tools/dotc/config/Feature.scala

```diff
@@ -36,7 +36,7 @@ object Feature:
   val into = experimental("into")
   val namedTuples = experimental("namedTuples")
   val modularity = experimental("modularity")
-  val betterMatchTypeExtractors = experimental("betterMatchTypeExtractors")
+  val betterMatchTypeExtractors = deprecated("betterMatchTypeExtractors")
```

Comment:
This is wrong, the language import is in `experimental`, not `deprecated`. This should be reverted.

Reply:
Fixed ✅